spec/controllers/gradings_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe Assessment::GradingsController do
+  let!(:course) { FactoryGirl.create(:course) }
+  let(:mission) { FactoryGirl.create(:mission, :with_general_questions, :with_coding_questions, :completed, course: course) }
+
+  describe 'GET new' do
+
+    before do
+      get :new, course_id: course.id,
+                mission_id: mission.id,
+                submission_id: mission.assessment.submissions.first.id
+    end
+
+    it 'assigns @summary' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.